srcpkgs/igt-gpu-tools/template

```diff
@@ -1,20 +1,30 @@
 # Template file for 'igt-gpu-tools'
 pkgname=igt-gpu-tools
-version=1.23
-revision=2
+version=1.24
+revision=1
 build_style=meson
-hostmakedepends="flex gtk-doc pkg-config swig"
-makedepends="cairo-devel libXrandr-devel libXv-devel libkmod-devel
- libunwind-devel procps-ng-devel python3-devel libressl-devel"
-short_desc="Tools for development and testing of the Intel DRM driver"
+configure_args="-Db_ndebug=false -Db_lto=false"
+# b_lto=true makes the build hang at a random point
+hostmakedepends="pkg-config flex peg python3-docutils gtk-doc"
+makedepends="libdrm-devel libkmod-devel procps-ng-devel libunwind-devel
+ elfutils-devel pixman-devel valgrind-devel cairo-devel xmlrpc-c-devel gsl-devel
+ alsa-lib-devel json-c-devel libXrandr-devel"
+short_desc="Tools for development and testing of the DRM drivers"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://gitlab.freedesktop.org/drm/igt-gpu-tools"
 distfiles="${XORG_SITE}/app/${pkgname}-${version}.tar.xz"
-checksum=4d4b086c513bace5c23d0889de3f42ac3ebd3d968c64dedae6e28e006a499ad0
+checksum=57357c72feeafc923c9cfd2d1234bd80e120fc7cc6099eac81158ec351a821bf
 
 lib32disabled=yes
-archs="i686 x86_64"
+archs="i686* x86_64*"
+
+build_options="tests"
+desc_option_tests="Build tests"
+
+if [ ! ${build_option_tests} ]; then
```

Comment:
This should check for XBPS_CHECK_PKGS instead (i.e check target).

Reply:
get rid of the build option and use:

```
if [ $XBPS_CHECK_PKGS ]]; then
 ...
fi
```

Reply:
done